exporters/webhook/app.py

```diff
@@ -117,24 +117,30 @@ async def prometheus_metric(received_metric: PelorusMetric):
 
     if received_metric_type == PelorusMetricSpec.COMMIT_TIME:
         in_memory_commit_metrics.add_metric(
-            metric.commit_hash, prometheus_metric, metric.timestamp
+            metric.commit_hash,
+            prometheus_metric,
+            metric.timestamp,
```

Comment:
remove trailing comma so only the line of deploytime metric is changed

Reply:
done

Reply:
failures still have trailing comma

Reply:
fixed in failure as well.